common/src/Microsoft.Azure.IIoT.Abstractions/src/IoTEdgeVariables.cs

```diff
@@ -0,0 +1,28 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Abstractions {
+
+    /// <summary>
+    /// Common IoT Edge runtime environment variables.
+    /// </summary>
+    public static class IoTEdgeVariables {
+
+        /// <summary> IoT Edge device id </summary>
+        public const string IOTEDGE_DEVICEID = "IOTEDGE_DEVICEID";
```

Comment:
These should be properties

Reply:
`IoTEdgeVariables` is not intended as a property or configuration class. It is only a collection of IoT Edge runtime environment variable names so that we do not use duplicate string literals in the code. This class also gives us the possibility to see where in the code IoT Edge environment variables are referenced. It serves the same purpose as `PcsVariable` collection for out `PCS_` environment variables.